core/internal/example/example.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+// package example provides a very simple example module for registering
+// a name with an owner.
+package example;
+
+import "cosmos/app/v1alpha1/module.proto";
+import "cosmos/msg/v1/msg.proto";
+
+message Module {
+  option (cosmos.app.v1alpha1.module) = {
+    go_import: "github.com/cosmos/cosmos-sdk/core/internal/example"
+    use_package: {
+      name: "example"
+    }
```

Comment:
Is this for app wiring? What does this mean?

Reply:
It's sort of informational for now, but eventually the idea is that it helps clients know about protobuf package revisions (for instance when we add comments like `Since 0.46` in .proto files). See https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/app/v1alpha1/module.proto#L25-L81.

Reply:
Who and where is going to check this? Do we really need this in proto? It seams it's Cosmos SDK Go implementation specific, so why not using Go directly rather than going through proto?